src/main/java/pl/osowicz/task_manager/Task/TaskController.java

```diff
@@ -74,37 +76,37 @@ public String showTasks(@RequestParam(required = false) Status status, Model mod
         }
         model.addAttribute("tasks", tasks);
         model.addAttribute("status", status);
-        return "taskList";
+        return "/task/list";
     }
 
-    @GetMapping("/editTask")
+    @GetMapping("/task/edit")
     public String editTask(@RequestParam(name = "id") Long id, Model model, Status status) {
         Optional<Task> task = taskRepository.findById(id);
         if (task.isPresent()) {
-            List<User> users = userRepository.findAll();
+            List<User> users = userRepository.findAllByActiveIsTrue();
             model.addAttribute("task", task.get());
             model.addAttribute("users", users);
             model.addAttribute("listStatus", status);
         } else {
             return "redirect:/";
         }
-        return "editTask";
+        return "task/edit";
     }
 
-    @PostMapping("/editTask")
+    @PostMapping("/task/edit")
     public String saveEditedTask(Task task, @RequestParam(name = "status", required = false) Status listStatus) {
         setTaskStatus(task);
         taskRepository.save(task);
         return redirectToPreviousTaskList(listStatus);
     }
 
-    @RequestMapping("/deleteTask")
+    @RequestMapping("/task/delete")
```

Comment:
Starajmy się nie używać RequestMapping bez zdefiniowania metody - bo wtedy będzie obsługiwała wszystkie metody. Tutaj raczej chcemy zrobić @DeleteMapping

Reply:
Przerobienie teo na @DeleteMapping zwraca błąd "There was an unexpected error (type=Method Not Allowed, status=405).
Request method 'GET' not supported". Z tego co widzę, HTML5 nie wspiera DELETE i PUT, (domyślnie wyłączone) a próba włączenia przez application.properties (jak tutaj https://stackoverflow.com/a/62174233 ) nie zdało egzaminu. Stąd @RequestMapping.

Reply:
A dobra, zapomniałem, że to jest używane w formatkach. To w takim przypadku i tak lepiej zamiast @RequestMapping to dać @PostMapping i w formularzu dać metodę na POST, bo aktualnie wysyłane jest GETem, co jest najgorszą z możliwych opcji jeżeli chodzi o obsługę usunięć